Tasks/Xcode/xcode.ts

```diff
@@ -231,11 +231,17 @@ async function run() {
             xcb.pipeExecOutputToTool(xcPrettyTool);
         }
 
-        //--- Xcode Build ---
-        await xcb.exec();
+        let xcodeTaskErrors;
+        try {
+            //--- Xcode Build ---
+            await xcb.exec();
+        } catch (err) {
+            xcodeTaskErrors = err;
+            tl.debug('Xcode task failed: ' + err);
+        }
 
         //--------------------------------------------------------
-        // Test publishing
+        // Test publishing - publish even if tests fail
```

Comment:
Might be better to check if the actions contains "test" and only then try to publish test results.